src/components/autocompleter/autocompleter.md

```diff
@@ -0,0 +1,31 @@
+The `Autocompleter` component is a wrapper for Select component with the additional functionality searching for a specific models from an API endpoint.
+
+`apiEndpoint` should be the route of the api's endpoint (excluding the base api), eg. `/workspaces`.
+`value` and `models` shape is expected to be an object(s) with an `id` key, and a display attribute that is one of the following: `title`, `name`, `full_name`, `currency`;
```

Comment:
If you describe the prop type in `Autocompleter.propTypes` then this would be auto-documented in the props table. 

I think you'll still need to add some text for the display attribute. But maybe it's possible to describe that as prop-types too. I'm not sure though.